examples/plugins/slack/Readme.md

```diff
@@ -0,0 +1,54 @@
+# Slack Integration
+Use this plugin to send slack notifications to users for approvals.
+
+## Requirements
+1. Setup a Slack App and get its bot user oauth token (from Slack API). It starts with `xoxb-`.
+2. Ensure it has proper permissions depending on if the channel is public or private. 
+
+- chat:write        # Must Have
+- channels:read     # For public channels
+- channels:write    # For public channels
+- groups:read       # For private channels
+- groups:write      # For private channels
+- users:read        # Must Have
+- users:read.email  # Must Have
+
+
+3. Create a slack channel and add the Application.
+- Should be under `Integrations` tab within Channel management.
+
+4. Add your slack bot token to the .env.production file and the Channel Name.
+```
+SLACK_BOT_TOKEN=<your-slack-bot-token>
+BASE_URL=<your-base-url>
```

Comment:
There's already going to be a Client Origin Env Var defined, so why not just use that?